scripts/update.sh

```diff
@@ -34,8 +34,8 @@ done
 
 # Pick a default rom if present
 if [ -z "$rom" ]; then
-  rom=$(crc32 *.nes | grep ^1bd39032 | head -1)
-  rom=$(echo ${rom#1bd39032})
+  rom=$(sha256sum *.nes | grep ^070d22fe | head -1)
```

Comment:
crc32 isn't default on many distros, so I swapped to sha256sum which is more common